dev/checkstyle/fileheader.txt

```diff
@@ -1,19 +1,23 @@
-/*
- * CDDL HEADER START
- *
- * The contents of this file are subject to the terms of the
- * Common Development and Distribution License (the "License").
- * You may not use this file except in compliance with the License.
- *
- * See LICENSE.txt included in this distribution for the specific
- * language governing permissions and limitations under the License.
- *
- * When distributing Covered Code, include this CDDL HEADER in each
- * file and include the License file at LICENSE.txt.
- * If applicable, add the following below this CDDL HEADER, with the
- * fields enclosed by brackets "[]" replaced with your own identifying
- * information: Portions Copyright [yyyy] [name of copyright owner]
- *
- * CDDL HEADER END
- */
+/\*
+ \* CDDL HEADER START
+ \*
+ \* The contents of this file are subject to the terms of the
+ \* Common Development and Distribution License \(the "License"\)\.
+ \* You may not use this file except in compliance with the License\.
+ \*
+ \* See LICENSE\.txt included in this distribution for the specific
+ \* language governing permissions and limitations under the License\.
+ \*
+ \* When distributing Covered Code, include this CDDL HEADER in each
+ \* file and include the License file at LICENSE\.txt\.
+ \* If applicable, add the following below this CDDL HEADER, with the
+ \* fields enclosed by brackets "\[\]" replaced with your own identifying
+ \* information: Portions Copyright \[yyyy\] \[name of copyright owner\]
+ \*
+ \* CDDL HEADER END
+ \*/
 
+/\*
+ \* (Portions )?Copyright \(c\) \d\d\d\d(, \d\d\d\d)?, [^0-9].*\.
+ \*/
+package .*;
```

Comment:
what is the purpose of this "package.*"?

Reply:
Basically it only prevents any blank lines between the end of copyright comment and the `package` definition.